tests/e2e/unified_test.go

```diff
@@ -81,7 +83,33 @@ func TestAllInSeries(t *testing.T) {
 		}
 	})
 
-	t.Run("fixtures", func(t *testing.T) {
+	testFixturesInSeries(ctx, t, "AllInSeries", false, cancel)
```

Comment:
Not a blocker, but I think it's an error that we think about samples & fixtures so differently.  It didn't matter when we were just calling RunCreateDeleteTest, but over time we've added to fixtures but not to samples (OK, it was me :-) )!  Maybe we preload the samples and fixtures into a common data structure, and then have a func to load the full data if needed in each test.  (parameterizing the project-id requires starting mockgcp, that's what we're trying to avoid doing at top level)